participants/notify.py

```diff
@@ -455,6 +494,11 @@ def handle_lifecycle_control(self, ctrl):
             okdirs = ctrl.config.get("notify", "allowed_attachment_dirs")
             self.allowed_attachment_dirs = okdirs.split()
 
+            email_whitelist = ctrl.config.get("notify", "email_whitelist")
```

Comment:
or just : self.email_whitelist = ctrl.config.get("notify", "email_whitelist").split()

Reply:
agree, but i wrote in the same style as it is

Reply:
It probably should also check that the config key exists first, as config.get() throws exception if it doesn't, if I recall right. So that it wouldn't break existing installations on upgrade, that don't have the white and black list in config

Reply:
added option check to not break existing installation